tailwind.config.js

```diff
@@ -4,7 +4,17 @@ export default {
   theme: {
     extend: {
       colors: {
-        /* Figma 색상 팔레트 정의 필요 */
+        primary: 'var(--primary)',
+        "primary-darken": 'var(--primary-darken)',
+        "primary-lighten": 'var(--primary-lighten)',
+        "primary-lighter": 'var(--primary-lighter)',
+        "accent1": 'var(--accent-1)',
+        "accent2": 'var(--accent-2)',
+        "accent3": 'var(--accent-3)',
+        "accent4": 'var(--accent-4)',
+        "accent5": 'var(--accent-5)',
+        "accent6": 'var(--accent-6)',
+        "accent7": 'var(--accent-7)',
```

Comment:
아래처럼 작성하는 건 어떨까요?

색상이 많지 않아서 지금처럼 작성해도 괜찮긴 하지만 구분하기 좋을 것 같아 제안드립니다 ㅎㅎ
```suggestion
        primary: {
          DEFAULT: 'var(--primary)',
          darken: 'var(--primary-darken)',
          lighten: 'var(--primary-lighten)',
          lighter: 'var(--primary-lighter)',
        },
        gray: {
          "accent1": 'var(--accent-1)',
          "accent2": 'var(--accent-2)',
          "accent3": 'var(--accent-3)',
          "accent4": 'var(--accent-4)',
          "accent5": 'var(--accent-5)',
          "accent6": 'var(--accent-6)',
          "accent7": 'var(--accent-7)',
```

Reply:
리뷰 감사합니다~! :)

제안해주신 내용이 더 좋아서 반영했어요~!